### PR TITLE
feat: test pr

### PR DESCRIPTION
**Pull request by VenusIM**
test controller create

feature/ijh

### CODE REVIEW

**Comment by yeomje1** [comment]
review test

**Comment by dmsp1004** [comment]
Good PR

File: petkin/src/main/java/com/project/petkin/test/TestController.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.project.petkin.test;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/v1")
+public class TestController {
+
```
Comment (gusdn0413) [inline petkin/src/main/java/com/project/petkin/test/TestController.java:10]: good

Comment (VenusIM) [reply petkin/src/main/java/com/project/petkin/test/TestController.java:10]: 131321232

**Comment by Hello5123** [comment]
hi

**Pull request merged**